_rules/SC1-2-2-video-audio-alternative.md

```diff
@@ -0,0 +1,109 @@
+---
+name: Vídeo audio alternative
```

Comment:
Vídeo --> Video

Reply:
@ShadowBB - could you push a fix for this, so we can merge.

Reply:
Going with "video has audio alternative"